### ISSUE TITLE
Build Error - No configuration files have been found in your configDir

### ISSUE THREAD

**Issue by drewbaker** [nuxt-community/storybook#259]
### Version

@nuxtjs/storybook: 
nuxt: 

### Steps to reproduce

```
npm install --save-dev @nuxtjs/storybook

Then:
npx nuxt storybook build
```

### What is Expected?

That the storybook gets built into a static directory.

### What is actually happening?

At static build, it crashes with the following error. Notice how it is complaining about `/.storybook` but I'd expect that should be `.nuxt-storybook`. Maybe that is the issue? 

The regular `npx nuxt storybook` runs fine, it's just the static build that is failing.

```
ERR! Error: No configuration files have been found in your configDir (/Users/drew/Sites/library-website-nuxt/.storybook).
ERR! Storybook needs either a "main" or "config" file.
ERR!     at validateConfigurationFiles (/Users/drew/Sites/library-website-nuxt/node_modules/@storybook/core-common/dist/cjs/utils/validate-c
onfiguration-files.js:50:11)
ERR!     at loadCustomPresets (/Users/drew/Sites/library-website-nuxt/node_modules/@storybook/core-common/dist/cjs/utils/load-custom-presets
.js:19:62)
ERR!     at loadAllPresets (/Users/drew/Sites/library-website-nuxt/node_modules/@storybook/core-common/dist/cjs/presets.js:310:105)
ERR!     at buildStaticStandalone (/Users/drew/Sites/library-website-nuxt/node_modules/@storybook/core-server/dist/cjs/build-static.js:95:48
)
ERR!     at async Object.buildStatic (/Users/drew/Sites/library-website-nuxt/node_modules/@storybook/core-server/dist/cjs/build-static.js:14
2:5)
ERR!  Error: No configuration files have been found in your configDir (/Users/drew/Sites/library-website-nuxt/.storybook).
ERR! Storybook needs either a "main" or "config" file.
ERR!     at validateConfigurationFiles (/Users/drew/Sites/library-website-nuxt/node_modules/@storybook/core-common/dist/cjs/utils/validate-c
onfiguration-files.js:50:11)
ERR!     at loadCustomPresets (/Users/drew/Sites/library-website-nuxt/node_modules/@storybook/core-common/dist/cjs/utils/load-custom-presets
.js:19:62)
ERR!     at loadAllPresets (/Users/drew/Sites/library-website-nuxt/node_modules/@storybook/core-common/dist/cjs/presets.js:310:105)
ERR!     at buildStaticStandalone (/Users/drew/Sites/library-website-nuxt/node_modules/@storybook/core-server/dist/cjs/build-static.js:95:48
)
ERR!     at async Object.buildStatic (/Users/drew/Sites/library-website-nuxt/node_modules/@storybook/core-server/dist/cjs/build-static.js:14
2:5)
```

My storybook config in nuxt.config.js is this:
```
storybook: {
        stories: ["~/stories/**/*.stories.js"],
        parameters: {
            layout: "fullscreen",
            backgrounds: {
                default: "default",
                values: [
                    {
                        name: "White",
                        value: "#ffffff",
                    },
                    {
                        name: "Grey",
                        value: "#F2F2F2",
                    },
                    {
                        name: "Dark blue",
                        value: "#032D5B",
                    },
                    {
                        name: "Blue",
                        value: "#0B6AB7",
                    },
                    {
                        name: "Light blue",
                        value: "#EAF2FB",
                    },
                ],
            },
        },
        webpackFinal(config, { configDir }) {
            // Allow webpack to auto-load .gql and .svg files
            config.resolve.extensions.push(".gql", ".svg")

            return config
        },
    },
```

**Comment by nielwijers** [comment]
I found a temporary workaround.
Setting the SBCONFIG_CONFIG_DIR to ./.nuxt-storybook/storybook works for me:
`env SBCONFIG_CONFIG_DIR=./.nuxt-storybook/storybook npx nuxt storybook build`

**Comment by lihbr** [comment]
Fixed: https://github.com/storybookjs/storybook/pull/14566 🙌 hopefully will get published in a few days~

**Comment by lihbr** [comment]
^ @farnabaz + https://github.com/storybookjs/storybook/releases/tag/v6.2.8 ☺️

**Comment by farnabaz** [comment]
Just released v4.0.2 with Storybook@6.2.8
Thank to @lihbr
